.vscode/launch.json

```diff
@@ -7,7 +7,7 @@
             "request": "launch",
             "justMyCode": false,
             "stopAtEntry": true,
-            "program": "${workspaceRoot}/debug/powershell",
+            "program": "${workspaceRoot}/debug/pwsh",
```

Comment:
This is being done in #5189. It shouldn't also be in this PR, I don't think.

Reply:
I'll remove it, had to change it locally to debug this.